docs/en/reference/architecture.rst

```diff
@@ -26,26 +26,26 @@ Doctrine ORM Packages
 
 Doctrine ORM is divided into three main packages.
 
--  Common
--  DBAL (includes Common)
--  ORM (includes DBAL+Common)
+-  Persistence
+-  DBAL
```

Comment:
As those are about separate packages that have their own docs, should we have links to their docs ?